modules/runtime/nuxeo-runtime-migration/src/main/java/org/nuxeo/runtime/migration/MigrationConstants.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour AL KOTOB
+ */
+package org.nuxeo.runtime.migration;
+
+/**
+ * @since 11.1
+ */
+public class MigrationConstants {
+
+    public static final String MIGRATION_ENTITY_TYPE = "migration";
+
+    public static final String MIGRATION_ID = "id";
+
+    public static final String MIGRATION_DESCRIPTION = "description";
+
+    public static final String MIGRATION_DESCRIPTION_LABEL = "description.label";
```

Comment:
Isn't a problem for javascript when it comes to reading this property (due to `.`)?

Reply:
👍 should be `descriptionLabel` as what we do on other writers.